lib/remote_page.py

```diff
@@ -191,6 +191,16 @@ def _fix_relative_urls(self, url, tag_name, attr_name):
                 elif not value.startswith('/'):
                     element[attr_name] = '../' + value
 
+                # If chapter is in a nested directory, folders names between
+                # the module and the chapter must be removed
+                # from the link since a-plus urls do not have them.
+                if len(element[attr_name].split('/')) > 3:
+                    path_to_module = ('/').join(element[attr_name].split('/')[0:3])+'/'
+                    # Links can be written with our without the ending slash ('/')
+                    index = 1 if element[attr_name].split('/')[-1] else 2
+                    chapter_name = element[attr_name].split('/')[-index]
```

Comment:
Maube `element[attr_name].rstrip('/').rpartition('/')[-1]`? Though, if we already have `path_parts`, we can do `chatper_name = path_parts[-1] or part_parts[-2]`